pkg/api/live/v1/router.go

```diff
@@ -53,6 +53,8 @@ func (r *MessageRouter) Message(byMsg []byte) error {
 		return r.MessageResponse(byMsg)
 	case interfaces.TypeMetadataResponse:
 		return r.MetadataResponse(byMsg)
+	case interfaces.TypeUtteranceEndResponse:
+		return r.UtteranceEndResponse(byMsg)
```

Comment:
Is appears that there is some byte stream that is returned from the Deepgram platform here. Can this be turned into a Go Struct?